File: sdk/dart/lib/model/category.dart
part of swagger.api;


@Entity()
class Category {
  
  @Property(name: 'ID')
  String ID = null;
  

  @Property(name: 'Name')
  String name = null;
  

  @Property(name: 'Description')
  String description = null;
  

  @Property(name: 'xp')
  Object xp = null;
  

  @Property(name: 'ListOrder')
  int listOrder = null;
  

  @Property(name: 'Active')
  bool active = null;
  

  @Property(name: 'ParentID')
  String parentID = null;
  
  Category();

  @override
  String toString()  {
    return 'Category[ID=$ID, name=$name, description=$description, xp=$xp, listOrder=$listOrder, active=$active, parentID=$parentID, ]';
  }

}

File: sdk/async-scala/src/main/scala/io/swagger/client/model/Category.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Category (
  ID: Option[String],
Name: Option[String],
Description: Option[String],
xp: Option[Any],
ListOrder: Option[Integer],
Active: Option[Boolean],
ParentID: Option[String]
)
